www/__tests__/enketoHelper.test.ts

```diff
@@ -63,7 +66,7 @@ it('gets the model response, if avaliable, or returns null', () => {
   };
 
   //if no xmlModel, returns null
-  expect(getInstanceStr(null, opts)).toBe(null);
+  expect(getInstanceStr('', opts)).toBe(null);
```

Comment:
Because omitting `xmlModel` would be the same as it being `undefined`, which is an invalid input for this function.
This function only does anything if `xmlModel` is truthy